csharp-basics/exercises/Polymorphism/DragRace.Tests/AudiTest.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace DragRace.Tests
+{
+    [TestClass]
+    public class AudiTest
+    {
+        [TestMethod]
+        public void SpeedUp_SpeedUpBy1_1Expected()
+        {
+            //Arrange
+            Audi aCar = new Audi();
+            //Act
+            var actual = aCar.SpeedUp();
+            var expected = 1;
+            //Assert
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestMethod]
+        public void SlowDown_SlowDownBy1_1Expected()
+        {
+            //Arrange
+            Audi aCar = new Audi();
+            aCar.SpeedUp();
+            aCar.SpeedUp();
+            //Act
+            var actual = aCar.SlowDown();
+            var expected = 1;
```

Comment:
Šis ir arrange

Reply:
Tad nesaprotu ar ko atšķirās act no arrange,
![image](https://user-images.githubusercontent.com/79905483/128902510-b2ac4a20-abdd-4415-8c45-72c69d328c3f.png)

Reply:
Bilde no lekcijas

Reply:
Tik un tā tas ir Arrange. Act liek tikai to, kur tiek veikta darbība priekš testa. Gan jau iepriekš kāds kļūdijās.